.github/workflows/publish-napi.yml

```diff
@@ -236,12 +240,9 @@ jobs:
 
       - name: Publish
         run: |
-          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
-          npm publish
```

Comment:
@amrbashir I don't understand the reason that we were calling publish twice. Any recollection?